src/compiler/diagnosticMessages.json

```diff
@@ -2356,6 +2356,10 @@
         "category": "Error",
         "code": 2513
     },
+    "Cannot use a negative integer for indexing on '{0}'.": {

```

Comment:
The current wording for this sort of bounds check is:

`Tuple type '{0}' of length '{1}' has no element at index '{2}'.`

So, maybe something like:

- `Tuple type cannot be indexed negatively`

Or just let it say "Tuple type ... of length ... has no element at index '-1'."?

@DanielRosenwasser

Reply:
> A tuple type cannot be indexed with a negative value.